backend/cmd/headlamp.go

```diff
@@ -929,7 +932,49 @@ func (c *HeadlampConfig) OIDCTokenRefreshMiddleware(next http.Handler) http.Hand
 	})
 }
 
+func initializeTelemetry(config *HeadlampConfig) {
+	telemetryCfg := telemetry.DefaultConfig()
```

Comment:
Would it make sense if this was moved within the pkg/telemetry package? Then it could be called from here.